daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -44,6 +44,8 @@ message Node {
         NodeExercise exercise = 5;
         NodeLookupByKey lookup_by_key = 6;
     }
+
+    string version = 7; // optional *since dev*
```

Comment:
We should probably leave a reference to the ticket for the rollout of the next DAML-LF version here?

Reply:
I this is part of the normal LF release process to traverse the following files to update the *dev* reference: 
`transaction.proto`, `value.proto`, `transaction.rst`, `value.proto`, `daml_lf_1.proto` and `daml-lf-1.rst`

For this reason I do not think a referent to the github issue is necessary here.